pkg/config/setup/config.go

```diff
@@ -469,6 +469,8 @@ func InitConfig(config pkgconfigmodel.Setup) {
 	config.BindEnvAndSetDefault("network_path.collector.pathtest_ttl", "15m")
 	config.BindEnvAndSetDefault("network_path.collector.pathtest_interval", "5m")
 	config.BindEnvAndSetDefault("network_path.collector.flush_interval", "10s")
+	config.BindEnvAndSetDefault("network_path.collector.reverse_dns_enrichment.enabled", true)
```

Comment:
Is `network_devices.netflow.reverse_dns_enrichment_enabled` being deprecated or is it going to remain supported going forward?

Reply:
It's going to remain supported going forward. The intent is that if `network_devices.netflow.reverse_dns_enrichment_enabled` || (`network_path.collector.reverse_dns_enrichment.enabled` && `network_path.connections_monitoring.enabled`) is true the component will be initialized and it's up to `Netflow` and `Network Path` to decided to use it.

Though in practice they won't overlap because Network Traffic paths is part of process-agent and Netflow is part of the regular agent.

Reply:
Great, thanks for the explanation!